src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -9,11 +9,19 @@
 
 namespace Microsoft.ML
 {
+    /// <summary>
+    /// Spcifies the names of output and input columns for a transformation.
```

Comment:
>Spcifies [](start = 8, length = 8)

some vowels got lost across whole file

Reply:
Good catch thank you!

---
In reply to: [263041582](https://github.com/dotnet/machinelearning/pull/2862#discussion_r263041582) [](ancestors = 263041582)